From f34703744a2fd47a4dfb2128c3eadb48b5500b65 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Thu, 9 May 2013 09:39:15 -0700
Subject: [PATCH 100/160] systemctl does not expand %u, so revert back to %I

The description field is only displayed by systemctl, and
it can't expand %u properly (it will always display "root").

(cherry picked from commit 9541fe6adff9941e487084c718ff2d46ed2929c6)
---
 units/user@.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/units/user@.service.in b/units/user@.service.in
index 3cf1347..ece671d 100644
--- a/units/user@.service.in
+++ b/units/user@.service.in
@@ -6,7 +6,7 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=User Manager for %u
+Description=User Manager for %I
 After=systemd-user-sessions.service
 
 [Service]
-- 
1.8.3.1

